models/dataset.go

```diff
@@ -11,7 +11,10 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
-const created = "created"
+const (
+	created               = "created"
```

Comment:
there are also state constants defined in api/dataset - is it worth centralising them and being explicit about which belong to datasets / versions / instances.